TypeCobol.Test/Parser/Programs/Cobol85/ProgramEndDisorder.rdz.cbl

```diff
@@ -0,0 +1,19 @@
+﻿       IDENTIFICATION DIVISION.
+       PROGRAM-ID. Main.
+      
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. Nested0.
+      
+       END PROGRAM Main.
+      
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. Nested1.
+      
+       END PROGRAM Nested1.
+      
+       END PROGRAM Nested0.
```

Comment:
According to me, 2 errors must be generated:
When `Main` is closed, every nested program is closed so `Nested0` is missing its `END PROGRAM` (see rules in #1162).
Then `END PROGRAM Nested0` is encountered and is attached to the root (this might be a wrong choice from me), which is not a program, so it generates an error.

This diagram represents the nodes and the errors.
![image](https://user-images.githubusercontent.com/76429957/111657679-f583b780-880b-11eb-859e-fdbab73b3ef5.png)

Reply:
OK your explanation is compliant with the initial issue.